openmc/element.py

```diff
@@ -319,3 +324,8 @@ def expand(self, percent, percent_type, enrichment=None,
             isotopes.append((nuclide, percent * abundance, percent_type))
 
         return isotopes
+
+
+class NoNaturalIsotopesWarning(Warning):
+    """Custom warning to indicate the absence of natural isotopes."""
+    pass
```

Comment:
Currently we don't use custom warning classes elsewhere in the code, so I'm reluctant to adopt it for this one particular use case as it would create an inconsistency. I would suggest removing this for now and starting a separate issue arguing for why custom warning classes are needed and what the approach should be going forward.